.editorconfig

```diff
@@ -9,19 +9,19 @@ trim_trailing_whitespace = true
 
 [*.go]
 indent_style = tab
-indent_size = 4
+indent_size = 8
 
-[*.tmpl]
-indent_style = tab
-indent_size = 2
-
-[Makefile]
+[*.{tmpl,html}]
 indent_style = tab
+indent_size = 4
```

Comment:
AFAIK all `.tmpl`-files use 2 spaces, should be "fixed" in this PR as well? (since the rules are changing, so should the files?)